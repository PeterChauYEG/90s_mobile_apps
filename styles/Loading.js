// react native
import { StyleSheet } from 'react-native'

const styles = StyleSheet.create({
  title: {
    fontSize: 20,
    textAlign: 'center',
    margin: 10
  }
})

export default styles
