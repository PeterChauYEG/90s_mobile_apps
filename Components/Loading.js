/**
 * @format
 * @flow
 */

// react
import React from 'react'

// react native
import { Text } from 'react-native'

// styles
import styles from './styles/Loading'

const Loading = (_: void) => {
  return <Text style={styles.title}>predicting...</Text>
}

Loading.propTypes = {}
Loading.defaultProps = {}

export default Loading
