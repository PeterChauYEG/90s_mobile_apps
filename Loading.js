/**
 * @format
 * @flow
 */

// react
import React from 'react'

// react native
import { Text } from 'react-native'

// styles 
import styles from './styles/Loading'

const Loading = () => {
  return <Text style={styles.title}>predicting...</Text>
}

export default Loading