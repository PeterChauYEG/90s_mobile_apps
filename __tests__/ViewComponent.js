import React from 'react'
import Loading from '../Loading'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(<Loading />).toJSON()
  expect(tree).toMatchSnapshot()
})
