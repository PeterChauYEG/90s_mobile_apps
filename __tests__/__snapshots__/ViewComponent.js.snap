// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Object {
      "fontSize": 20,
      "margin": 10,
      "textAlign": "center",
    }
  }
>
  predicting...
</Text>
`;
